packages/utils/src/RouterParams/RouterParams.js

```diff
@@ -4,14 +4,30 @@ import { withRouter, matchPath } from 'react-router-dom';
 import { connect } from 'react-redux';
 import isEqual from 'lodash/isEqual';
 
-export const routerParams = (Component) => {
-  class RouterParams extends React.Component {
-    componentDidMount() {
-      const {
-        match: { params, path, url },
-        onPathChange,
-        location,
-      } = this.props;
+class RouterParams extends React.Component {
```

Comment:
Can we create a JIRA ticket to make this component as function component?

Reply:
Yeah we should.